scripts/run-browsers-smoketests.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+const shell = require("shelljs");
+
+let result = 0;
+
+console.log(`
+Test webextension-polyfill on real browsers
+===========================================`);
+
+// Enable headless mode (currently only used when running on Firefox
+// because Chrome doesn't currently support the extensions in headless mode)
+process.env.HEADLESS = 1;
+
+console.log(`
+Runing smoketests on Chrome`);
```

Comment:
Typo: Runing -> Running

(Besides that, personally I would prefer a \n instead of a template string that starts with a new line, and same comment also for the similar template strings in run-module-bundlers-smoketests.js).